### PR TITLE
Makes the scream emote sound use a species proc rather than being a long list

### PR DESCRIPTION
**Pull request by SgtHunk**
## About The Pull Request

Currently, the scream emote takes its sounds from a LONG else if statement that includes each species that has a scream emote. With this pull request, species will now have a proc that will return the scream sound. This changes no functionality at all.

## Why It's Good For The Game

idk something tells me that it's good

## Changelog

Not player-facing.

### CODE REVIEW

File: code/modules/mob/living/carbon/human/species_types/humans.dm [hunk 1]
```diff
@@ -20,3 +20,19 @@
 	human.hairstyle = "Business Hair"
 	human.hair_color = "#bb9966" // brown
 	human.update_hair()
+
+/datum/species/human/get_scream_sound(mob/living/carbon/human/human)
```
Comment (Ghommie) [inline code/modules/mob/living/carbon/human/species_types/humans.dm:24]: i think this should be the default definition of `get_scream_sound` as a fallback for all those species that don't have scream sounds.

Comment (SgtHunk) [reply code/modules/mob/living/carbon/human/species_types/humans.dm:24]: I'll leave that to a maintainer's decision. As it is right now - species not mentioned in the scream emote's proc won't make any sound while screaming. Unless a maintainer wants me to change it, that functionality is being kept the way it is, as my goal was to simply improve the code, without changing the surface of how it plays out

**Comment by KittyNoodle** [comment]
scream code before legit scares me as to how bad it was

**Comment by RaveRadbury** [comment]
Does this mess with the mime scream emote override?

**Comment by Ghommie** [comment]
No, you can see the early return is still there.

**Comment by ShizCalev** [comment]
Does scream check if the mob has silent set, or if they have a tongue present?

**Comment by SgtHunk** [comment]
> Does scream check if the mob has silent set, or if they have a tongue present?

The scream's sound currently only checks if either

1- You're not a human.
2- Your mind is "miming".
3- You're not forcing it.

So, if you're a human,  you're not on a vow of silence, and the scream was forced rather than done voluntarily with *scream, it will play a sound corresponding to your species. (So if your species has no sound assigned, it'll be a silent scream)

File: code/modules/mob/living/carbon/human/species_types/humans.dm (hunk 1 above)
Comment (Mothblocks) [reply code/modules/mob/living/carbon/human/species_types/humans.dm:24]: You got it right

**Pull request merged**